src/main/drivers/bus_i2c.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *           
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *  
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */ 
+
+#include <stdbool.h>
+#include <stdint.h> 
+#include <stdlib.h>
+    
+#include <platform.h>
+
+#ifdef USE_I2C
+
+#include "io.h"
+#include "system.h"
+
+#include "config/config_master.h"
```

Comment:
Drivers should not have an upward dependency on `config_master.h`. Remove this and apply changes indicated below.